games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="3D open-source go-kart racing game"
+DESCRIPTION="3D open-source go-kart racing game with an extensive variety of characters, tracks, and modes to play."
```

Comment:
I think that the description could look better. (I would like to work on that.)

Reply:
@panos - Check with @diversys  or @humdingerb on HDS repo for game icon, pictures, and description concerns.
Example: https://depot.haiku-os.org/#!/pkg/0ad/haikuports/0/0/23b/alpha/5/x86_64?bcguid=bc367-NDUL

Reply:
@panos - I cleaned it up but go ahead and make it more official.